crypto/keyring/doc.go

```diff
@@ -21,20 +21,20 @@
 //
 // The backends:
 //
-//	os		The instance returned by this constructor uses the operating system's default
```

Comment:
the whitespace changes look odd but they render better

Before | After
--- | ---
<img width="896" alt="before" src="https://user-images.githubusercontent.com/3699047/201194933-f0f96f57-b98f-48ca-b49b-6afe4cca5133.png"> | <img width="838" alt="after" src="https://user-images.githubusercontent.com/3699047/201194861-e621129d-2505-4f8a-8de9-e994a7f62cdb.png">